op-chain-ops/cmd/celo-migrate/continuity_test.go

```diff
@@ -10,28 +10,32 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func makeRange(start int, bodies, receipts, tds, hashes, encodedHeaders [][]byte) *RLPBlockRange {
-	return &RLPBlockRange{
-		start:    uint64(start),
-		hashes:   hashes,
-		headers:  encodedHeaders,
-		bodies:   bodies,
-		receipts: receipts,
-		tds:      tds,
-	}
-}
+var (
+	hashes   = [][]byte{[]byte("hash0"), []byte("hash1"), []byte("hash2"), []byte("hash3")}
+	bodies   = [][]byte{[]byte("body0"), []byte("body1"), []byte("body2"), []byte("body3")}
+	receipts = [][]byte{[]byte("receipt0"), []byte("receipt1"), []byte("receipt2"), []byte("receipt3")}
+	tds      = [][]byte{[]byte("td0"), []byte("td1"), []byte("td2"), []byte("td3")}
 
-func TestCheckContinuity(t *testing.T) {
-	hashes := [][]byte{[]byte("hash0"), []byte("hash1"), []byte("hash2"), []byte("hash3")}
-	bodies := [][]byte{[]byte("body0"), []byte("body1"), []byte("body2"), []byte("body3")}
-	receipts := [][]byte{[]byte("receipt0"), []byte("receipt1"), []byte("receipt2"), []byte("receipt3")}
-	tds := [][]byte{[]byte("td0"), []byte("td1"), []byte("td2"), []byte("td3")}
-	decodedHeaders := []*types.Header{
+	decodedHeaders = []*types.Header{
 		{Number: big.NewInt(0), ParentHash: common.Hash{}},
 		{Number: big.NewInt(1), ParentHash: common.BytesToHash(hashes[0])},
 		{Number: big.NewInt(2), ParentHash: common.BytesToHash(hashes[1])},
 		{Number: big.NewInt(3), ParentHash: common.BytesToHash(hashes[2])},
 	}
+)
+
+func makeRange(start int, bodies, receipts, tds, hashes, encodedHeaders [][]byte) *RLPBlockRange {
+	r := &RLPBlockRange{}
+	r.start = uint64(start)
+	r.hashes = append(r.hashes, hashes...)
```

Comment:
Why is it necessary to use append here?

Reply:
I split the test into two, since CheckContinuity does not check the lengths any more, so I needed to move the variables into the global scope to make them accessible to both tests. Then to avoid having the tests modify the global variables ( the invalid parent hash case) I did this to ensure that make range copies the underlying array.